pages/listing/_title.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div>
+    <b-container class="py-5">
+      <b-row>
+        <b-col md="9">
```

Comment:
```suggestion
        <b-col md="7">
```
This is more consistent with the site layout